client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -63,131 +68,162 @@ class TransferDomainPrecheck extends React.PureComponent {
+				},
+			}
 		);
+		const buttonText = translate( 'I can access this email address' );
 
-		return this.getSection( heading, message, explanation, 2 );
+		return this.getSection( heading, message, buttonText, 2 );
 	}
 
 	getEppMessage() {
 		const { translate } = this.props;
 
 		const heading = translate( 'Get a domain authorization code.' );
 		const message = translate(
-			"You'll need this code to okay the transfer. " +
-				"We'll send you and email with a link to the place where you'll need to enter it."
-		);
-		const explanation = translate(
-			'A domain authorization code is a unique code linked only to your domain - kind of like a ' +
-				'password for your domain. Log in to your current domain provider to get one. ' +
-				"We call it a domain authorization code, but it's also called a secret code, auth code or EPP code."
+			'A domain authorization code is a unique code linked only to your domain — kind of like a ' +
+				"password for your domain. Log in to your current registrar to get one. We'll send you an email " +
+				'with a link to enter it and officially okay the transfer. We call it a domain authorization code, ' +
+				'but it might be called a secret code, auth code, or EPP code. {{a}}Learn more{{/a}}.',
+			{
+				components: {
+					a: <a href="#" rel="noopener noreferrer" />,
+				},
+			}
 		);
+		const buttonText = translate( 'I have my authorization code' );
+
+		return this.getSection( heading, message, buttonText, 3 );
+	}
+
+	getHeader() {
+		const { translate, domain } = this.props;
 
-		return this.getSection( heading, message, explanation, 3 );
+		return (
+			<Card compact={ true } className="transfer-domain-step__title">
+				<FormattedHeader
+					headerText={ translate( "Let's get {{strong}}%(domain)s{{/strong}} ready to transfer.", {
+						args: { domain },
+						components: { strong: <strong /> },
+					} ) }
+					subHeaderText={ translate(
+						'Log into your current registrar to complete a few preliminary steps.'
+					) }
+				/>
+				<img
+					className="transfer-domain-step__illustration"
+					src={ '/calypso/images/illustrations/migrating-host-diy.svg' }
+				/>
+			</Card>
+		);
 	}
 
 	render() {
 		const { translate } = this.props;
-		const headerLabel = translate(
-			"Let's get your domain ready to transfer. " +
-				'Log into your current registrar to complete a few preliminary steps.'
-		);
+		const { unlocked, currentStep } = this.state;
 
 		return (
 			<div className="transfer-domain-step__precheck">
-				<Card compact={ true }>{ headerLabel }</Card>
+				{ this.getHeader() }
 				{ this.getStatusMessage() }
 				{ this.getPrivacyMessage() }
 				{ this.getEppMessage() }
 				<Card className="transfer-domain-step__continue">
 					<div className="transfer-domain-step__continue-text">
 						<p>
 							{ translate(
-								'Note: These changes can take up to 20 minutes to take effect. ' +
+								'Note: These changes can take some time to take effect. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **1** time:
`translate( 'Note: These changes can take up to 20 minutes to take effect. Need help? {{a}}Get in touch with one of our Happiness Engineers{{/a}}.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).